commcare_connect/templates/opportunity/add_visits_existing_users.html

```diff
@@ -5,10 +5,25 @@
 {% block title %}{{ request.org }} - {{ opportunity.name }}{% endblock %}
 
 {% block content %}
-  <div class="container">
-    <h1 class="display-5">Edit Opportunity</h1>
-    <h4>Adding Visits for Existing Users</h4>
-    <form method="post"
+  <nav class="mt-2 small" aria-label="breadcrumb">
```

Comment:
A lot of these pages just add breadcrumbs. Is there a way to add breadcrumbs to the base template so we don't have to add them each time?